.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,45 +3,50 @@
 
 :wave: Just a quick reminder that this is a public repo. Please don't include any internal links or sensitive data (like PII, private code, customer names, site URLs, etc. Any fixes related to security should be discussed with Platform before opening a PR. If you're not sure if something is safe to share, please just ask!
 
-### BEFORE YOU PROCEED!!
-
-If you’re editing a feature without changing the spirit of the implementation, fixing bugs, or performing upgrades, then please proceed!
-
-If you’re adding a feature or changing the spirit of an existing implementation, please create a proposal in Cantina P2 using the MU Plugins Proposal Block Pattern. Please mention the [CODEOWNERS](.github/CODEOWNERS) of this repository and relevant stakeholders in your proposal :). Please be aware that any unplanned work may take some time to get reviewed. Thank you 🙇‍♀️🙇!
-
 ## For external contributors!
 
 Welcome! We look forward to your contribution! ❤️
 -->
 ## Description
 <!--
-A few sentences describing the overall goals of the Pull Request.
+A few sentences providing more context and describing the overall goals of the Pull Request.
 
-Should include any special considerations, decisions, and links to relevant GitHub issues.
-
-Please don't include internal or private links :)
+Should include any special considerations, decisions, and links to relevant publicly available sources (e.g. GitHub issues)
 -->
 
 ## Changelog Description
-<!--
-A description of the context of the change for a changelog. It should have a title, examples (if applicable), and why the change was made.
 
-**Please keep the changelog title format same as in example below (### <Title>), as this is later used to generate the changelog entry title.**
+<!-- Changelogs are published for our customers. Well-written entries help them stay informed on platform changes and all of the great work that we do! -->
+
+<!-- Write a concise description of changes in the relevant section.
+- Add new line items as needed.
+- Entries should follow the [Common Changelog Style Guide](https://github.com/vweevers/common-changelog). 
+- Remove all unused sections before merging.
+- Proof-read.
+-->
 
-Example for a plugin upgrade:
+### Added
+- <!-- e.g. "Added a new set of filters for MFA status" -->
+- <!-- e.g. "Dev-env: Added PHP 8.3 image" -->
 
-### Plugin Updated: Jetpack 9.2.1
+### Removed
+- <!-- e.g. "Dropped support of Node.js 14" -->
+- 
+
+### Fixed
+- <!-- e.g. "Fixed a bug causing blank lines in content to be ignored when using the Regex Parser" -->
+
+### Changed
```

Comment:
will we try to collate these in the changelog generator so that everything "removed" is under the same heading in the finished changelog?

Reply:
It's a very good question and probably we should!